series/liberec-pyvo/events/2020-03-24-seste.yaml

```diff
@@ -0,0 +1,16 @@
+city: liberec
+start: 2020-03-24 18:00:00
+name: Liberecké Pyvo
+number: 6
+venue: depo
+description: |
+  Je tu šesté Liberecké Pyvo, programátorský sraz nejen o Pythonu. Ač se akce koná pod záštitou Python komunity, vítáni jsou programátoři a admini používající jakýkoli (i žádný) programovací jazyk.
+
+  Abychom věděli, s kolika lidmi můžeme počítat, prosím **[nahlaste svou účast](https://doodle.com/poll/88tkfiun8t6bunah)**. Díky!
+
+talks:
+  - title: "Bude upřesněno"
+    description: |
+            Bude upřesněno
+    speakers:
+      - Bude upřesněno
```

Comment:
Ahoj, Travis se nějak zasekl, tak navrhuji, abys poslal další commit, bez těch description a speakers – není to povinné a je zbytečné tam psát nějaké placeholdery, když není žádný lepší text. Snad to taky prošťouchne Travise

Reply:
ok tak takto